datadog/util/config.py

```diff
@@ -130,31 +130,3 @@ def get_config(cfg_path=None, options=None):
         raise CfgNotFound
 
     return agentConfig
-
-
-def get_pkg_version():
-    """
-    Resolve `datadog` package version.
-    """
-    if not pkg:
-        return u"unknown"
-
-    dist = pkg.get_distribution("datadog")
-    # Normalize case for Windows systems
-    dist_loc = os.path.normcase(dist.location)
-    here = os.path.normcase(__file__)
-    if not here.startswith(dist_loc):
-        # not installed, but there is another version that *is*
-        raise pkg.DistributionNotFound
-
-    return dist.version
-
-
-def get_version():
-    """
-    Resolve `datadog` package version.
-    """
-    try:
-        return get_pkg_version()
-    except pkg.DistributionNotFound:
-        return u"Please install `datadog` with setup.py"
```

Comment:
I would keep these two functions for compatibility, in case some people use it in their code. 
Just make them return the `__version__` variable, and maybe add a comment in the docstring we're keeping them for compatibility but user should import `__version__` from `version` instead.

Reply:
👍 done